app/models/corporate_information_page.rb

```diff
@@ -244,4 +245,20 @@ def unique_worldwide_organisation_and_page_type
       errors.add(:base, "Another '#{display_type_key.humanize}' page was already published for this worldwide organisation")
     end
   end
+
+  def unique_editionable_worldwide_organisation_and_page_type
+    duplicate_scope = CorporateInformationPage
+                        .joins(:document_corporate_information_page)
+                        .where("document_corporate_information_pages.owning_document_id = ?", document_corporate_information_page.owning_document_id)
+                        .where(corporate_information_page_type_id:)
+                        .where("state not like 'superseded'")
```

Comment:
Could the SQL be upper cased, just to make it clearer this is SQL and not some rails thing, e.g. `.where("state NOT LIKE 'superseded'")`?